urllib3/response.py

```diff
@@ -351,3 +361,71 @@ def readinto(self, b):
         else:
             b[:len(temp)] = temp
             return len(temp)
+
+    def read_chunked(self, amt=None):
+        # FIXME: could this be handled better?
+        assert self.chunked
```

Comment:
I don't. =) From [the docs](https://docs.python.org/2/library/exceptions.html#exceptions.ValueError):

> Raised when a built-in operation or function receives an argument that has the right type but an inappropriate value, and the situation is not described by a more precise exception such as IndexError.

`TypeError` is closer:

> Raised when an operation or function is applied to an object of inappropriate type. The associated value is a string giving details about the type mismatch.

But altogether I recommend adding a new urllib3 exception that multiply-inherits from `urllib3.exceptions.ProtocolError` and `TypeError`.